model/qti/ParserFactory.php

```
@@ -1589,6 +1616,34 @@ private function buildSimpleFeedbackRule($subtree, $conditionName, $comparedValu
         return $feedbackRule;
     }
 
+
+    protected function buildScoreFeedbackRule($subtree, $conditionName, $comparedValue, $responseIdentifier, $xml)
+    {
+        $feedbackOutcomeIdentifier = (string) $subtree->responseIf->setOutcomeValue['identifier'];
```

PHPMD (2.15.0)

Avoid excessively long variable names like $feedbackOutcomeIdentifier. Keep variable name length under 20. (Naming Rules)

(LongVariable)